cypress/e2e/patient_spec/encouter-notes.cy.ts

```diff
@@ -0,0 +1,78 @@
+import { FacilityCreation } from "@/pageObject/facility/FacilityCreation";
+import { PatientEncounter } from "@/pageObject/Patients/PatientEncounter";
+
+const facilityCreation = new FacilityCreation();
+const patientEncounter = new PatientEncounter();
+
+describe("Encounter Notes - Threads, Messages, and Notes", () => {
+  beforeEach(() => {
+    cy.loginByApi("devdoctor"); // logging in as doctor
+    cy.visit("/");
+    facilityCreation.selectFacility("GHC Trikaripur");
+  });
+
+  it("create multiple threads and add messages", () => {
+    patientEncounter
+      .navigateToEncounters()
+      .openFirstEncounterDetails()
+      .openNotesTab()
+      
+
+    // first Thread
+    cy.contains("button", "New Thread").click();
+    cy.get("input[placeholder='Enter discussion title...']").type("Test Discussion final");
+    cy.contains("button", "Create").click();
+    cy.get("form").within(() => {
+      cy.get("textarea").type("This is a test message fianl!");
```

Comment:
```suggestion
      cy.get("textarea").type("This is a test message fianl!");
```

use combination of single line message as well as multi line text as well